java-manta-it/src/test/java/com/joyent/manta/client/helper/IntegrationTestHelper.java

```diff
@@ -73,10 +96,24 @@ public static void createTestBucketOrDirectory(final MantaClient client,
                                       final String testPath,
                                       final String testType) throws IOException {
         if ("buckets".equals(testType)) {
+            final String bucketPath = parseBucketPath(testPath);
             //Create bucket here
+            try {
+                client.createBucket(bucketPath);
+            } catch (MantaClientHttpResponseException e) {
+                if (e.getServerCode().equals(MantaErrorCode.BUCKET_EXISTS_ERROR)) {
+                    LOG.error("Test Bucket named {} already exists leading to {}"
```

Comment:
This isn't an `ERROR` level for logging. This is perhaps a `DEBUG` or `INFO` level message.

Reply:
Yes realized that since we want the `LOG` to provide information under regular circumstances.